_docs/tasks/installing-istio.md

```diff
@@ -1,19 +0,0 @@
----
-title: Installing Istio
-overview: This task shows you how to setup the Istio service mesh for different platforms.
-
-order: 10
-
-layout: docs
-type: markdown
----
-
-{% include home.html %}
-
-This task describes the steps needed for istio installation on various platforms.
-
-* [Quick Start - Kubernetes Platforms]({{home}}/docs/tasks/install-kubernetes.html).
```

Comment:
as I said a couple times, can we keep that page with links into `setup/` instead of `tasks/`

Reply:
I added redirect_from .. Please see the header. This is what we follow for all the blog articles as well. Any external links will get redirected to the correct page.

Reply:
:+1: missed that in the new file/didn't know that feature in jekyll